woocommerce/wc-admin
# Typescript conversion of DynamicForm Component

Fixes #6971 

Simple typescript conversion for the `<SettingsForm />` component. Some of the types created actually are for the `<Form />` component, so the argument could be made to continue and convert that component as well. 

### Detailed test instructions:

-   No changes in behavior
- Checkout branch
- Enable remote payments feature flag
- Activate the remote payment tester, and run `npm install` and `npm start`
- Navigate to the payment task and click on one of the sample configs (payfast). 
- You should see the fields associated with the selected gateway, generated by the `<SettingsForm />` component.
